src/main/webapp/app/overview/exercise-chatbot/exercise-chatbot.component.scss

```diff
@@ -0,0 +1,13 @@
+.chatbot-button {
+    position: fixed;
+    bottom: 80px; /* adjust this value to move the button up or down */
+    right: 30px;
+    z-index: 9999;
+}
+
+.custom-circle-button {
```

Comment:
Why don't you simply use the existing `btn-circle` for this?

Reply:
updated